File: cpp/2591.cpp
#include <iostream>
#include <string>

using namespace std;

void hamekameka(string word)
{
    int total = 0, first = 0, len = word.length(), i;
    bool flag = true;

    for (i = 0; i < len; i++) {
        if (word[i] == 'a') {
            total++;
        }

        if (flag && word[i] == 'a') {
            first++;
        }

        if (flag && word[i + 1] != 'a') {
            flag = false;
        }
    }

    cout << 'k' << string(first * (total - first), 'a') << '\n';
    return;
}

int main()
{
    int n;
    cin >> n;
    string w;

    for (n; n > 0; n--) {
        cin >> w;
        hamekameka(w);
    }

    return 0;
}

File: py/2591.py
def hamekameka(word):
    total = 0
    for i in range(len(word)):
        if word[i] == 'a':
            total += 1
        
    firstSyllabe = 0
    for i in range(len(word)):
        if word[i] == 'a':
            firstSyllabe += 1
            
        if word[i + 1] != 'a':
            break
        
    print('k' + (firstSyllabe * (total - firstSyllabe)) * 'a')

n = int(input())
for i in range(n):
    hamekameka(input())